apps/ledger-live-mobile/src/components/TabBar/TransferDrawer.tsx

```diff
@@ -212,6 +217,27 @@ export default function TransferDrawer({ onClose }: ModalProps) {
           {...IMAGE_PROPS_SMALL_NANO}
         />
       )}
+      {readOnlyModeEnabled && hasOrderedNano && (
```

Comment:
The component below is duplicated in several files, what do you think of putting it in a separate component to avoid code repetitions ?

Reply:
Good idea!